.cirrus.yml

```diff
@@ -0,0 +1,101 @@
+# release CI for FreeBSD
+compute_engine_instance:
+  image_project: freebsd-org-cloud-dev
+  image: family/freebsd-13-1
+  platform: freebsd
+  disk: 100 # Gb
+
+build_task:
+  timeout_in: 120m
+  only_if: $CIRRUS_TAG != ''
+  env:
+    AWS_ACCESS_KEY_ID: ENCRYPTED[dc5896620ebc12e98e6bbe96f72c5a2fe3785f439b7b2346797355f8d329a4bfd8ef6e58086bfc014be0d914424101cd]
+    AWS_SECRET_ACCESS_KEY: ENCRYPTED[6501cd594aca08c6c67cc679dd6f6d30db0cd44a81cceddebf32bb3d0a37f9af19cd71ddb7169d3f7b284a7829969f9e]
+    S3_HOST: ENCRYPTED[d3fef1b5850e85d80dd1684370b53183df2218f2d36509108a2703371afd9ebd3f9596ad4de52487c15ea29baed606b7]
+    TARBALL_EXT: "tar.xz"
+    ARCH: 64
+    ARTIFACT: "x86_64-freebsd"
+    DISTRO: "na"
+    RUNNER_OS: "FreeBSD"
+    ADD_CABAL_ARGS: "--enable-split-sections"
```

Comment:
I don't think this needs further documentation. It follows the existing gitlab workflow: https://gitlab.haskell.org/haskell/haskell-language-server/-/blob/master/.gitlab/ci.sh#L48

The GHC user guide documents the flag: https://downloads.haskell.org/~ghc/9.2.5/docs/html/users_guide/phases.html?highlight=split%20sections#ghc-flag--split-sections

Reply:
> I also wonder whether it would be better to pass this via creating a cabal.project.local rather than the ADD_CABAL_ARGS variable?

Definitely not. `cabal.project.local` is meant for local developer settings.